e2e-tests/IIoTPlatform-E2E-Tests/deploy/ConfigurationModelEx.cs

```diff
@@ -0,0 +1,76 @@
+﻿// ------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All rights reserved.
+//  Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+namespace IIoTPlatform_E2E_Tests.deploy {
+    using Microsoft.Azure.Devices;
+
+    /// <summary>
+    /// Configuration model extensions
+    /// </summary>
+    public static class ConfigurationModelEx {
```

Comment:
>ConfigurationModelEx [](start = 24, length = 20)

I think the whole class can be removed, we don't need different Models in our use case, we just create the one that is used to create the layered Deployments

Reply:
removed all the configuration model classes